app/views/errors/not_found.html.slim

```diff
@@ -1,14 +1,9 @@
 #error
   .error
     .error-num
-      h3
-        | Oops! Page not found
+      h3 #{t 'not_found.oops_title'}
```

Comment:
i asked you to add brackets t(....) it's more readable

Reply:
ohhh, i thought about another. I`ll fix it

Reply:
Done